packages/api/endpoints/granules.js

```diff
@@ -569,20 +580,32 @@ async function bulkOperations(req, res) {
   return res.status(202).send(asyncOperation);
 }
 
+const BulkDeletePayloadSchema = z.object({
```

Comment:
@marchuffnagle  I feel like there should be a better way to organize these.     It's a net positive on this PR, but I think we should debate a structure if we're going to wholesale do this across the api package.

Reply:
Sounds like a good architecture meeting topic.